appveyor.yml

```diff
@@ -40,6 +40,10 @@ environment:
       PYTHON: "3.5"
       ARCH: "64"
       CONFIGURATION: "Release"
+    - JOB: "Static_Crt_Build"
```

Comment:
@wesm number of new Appveyor's build jobs reduced to 1 and contain building only of C++ lib, but build time, unfortunately, is not reduced according to what Appveyor build results show.
Also have created [PR](https://github.com/lz4/lz4/pull/424) to patch Lz4 lib directly in origin repo.